packages/turf-clone/index.js

```diff
@@ -70,12 +77,21 @@ function cloneFeature(feature) {
  * @returns {FeatureCollection<any>} cloned Feature Collection
  */
 function cloneFeatureCollection(geojson) {
-    return {
-        type: 'FeatureCollection',
-        features: geojson.features.map(function (feature) {
-            return cloneFeature(feature);
-        })
-    };
+    const cloned = {type: 'FeatureCollection'};
+    // Add id & bbox first
+    if (geojson.id) cloned.id = geojson.id;
+    if (geojson.bbox) cloned.bbox = geojson.bbox;
+
+    // Custom Properties
+    Object.keys(geojson).forEach(function (key) {
```

Comment:
@DenisCarriere doesn't this in practice replace the [`cloneAll` feature](https://github.com/Turfjs/turf/pull/904/files#diff-367ea2e548de0355e3fcdf367c5aa64eR20)?